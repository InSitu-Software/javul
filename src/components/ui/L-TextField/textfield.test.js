import { mount } from 'vue-test-utils'
import Textfield from './textfield'

describe('Textfield', () => {

  test('is a Vue instance', () => {
    const wrapper = mount(Textfield)
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  test('renders a input', () => {
    const wrapper = mount(Textfield)
    expect(wrapper.contains('input')).toBe(true)
  })
})
