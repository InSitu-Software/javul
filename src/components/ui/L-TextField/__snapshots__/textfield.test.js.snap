// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Textfield should match the snapshot 1`] = `
<div>
  <label>
    
  </label>
   
  <input />
</div>
`;
